### PR TITLE
Cancel any action and remove any toast on logout

### LINKED ISSUE #2062: Clear Upload/Download progress toasts on logout
**Describe the bug**
Upload/Download progress toast remains if user logs out

**To Reproduce**
1. Login and start an upload (good to have a number of files)
2. Click on 'Logout'
3. The Upload/Download progress toast remains

**Expected behavior**
Upload/Download should be cancelled and progress toast cleared on logout.

### PR DESCRIPTION
**Pull request by Tbaut**
closes #2062

https://user-images.githubusercontent.com/33178835/161984242-a254364b-b697-4a2b-89e4-6000d3830d3f.mp4

### CODE REVIEW

File: packages/files-ui/src/Components/Elements/Notifications/NotificationList.tsx [hunk 1]
```diff
@@ -58,6 +58,10 @@ const NotificationList = ({ notifications }: INotificationListProps) => {
   const thisWeeksNotifications = notifications.filter(n => dayjs(Date.now()).diff(dayjs.unix(n.createdAt), "day") <= 7)
   const olderNotifications = notifications.filter(n => dayjs(Date.now()).diff(dayjs.unix(n.createdAt), "day") > 7)
 
+  useEffect(() => {
+    console.log("notif", notifications.length)
+  }, [notifications])
+
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Elements/Notifications/NotificationList.tsx]:
```suggestion
```

File: packages/files-ui/src/Components/Elements/Notifications/NotificationList.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React from "react"
+import React, { useEffect } from "react"
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Elements/Notifications/NotificationList.tsx]:
```suggestion
import React from "react"
```

File: packages/files-ui/src/Components/Layouts/AppNav.tsx [hunk 3]
```diff
@@ -260,11 +261,14 @@ const AppNav = ({ navOpen, setNavOpen }: IAppNav) => {
   const { publicKey, isNewDevice, shouldInitializeAccount, logout } = useThresholdKey()
   const { removeUser, getProfileTitle, profile } = useUser()
   const location = useLocation()
+  const { removeAllToasts } = useToasts()
 
   const signOut = useCallback(() => {
+    console.log("hop")
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Layouts/AppNav.tsx]:
```suggestion
```

**Review by asnaith — APPROVED** [review]

**Comment by asnaith** [comment]
Working well 👍

During testing, I realized on mobile that it's not possible to sign-out without canceling the upload process first because the element is covered. Personally, I think this is ok as-is but wanted to mention it in case anyone thinks differently about it (we can create a new issue if there's suggestions for an improvement here). 

Screen Shot 2022-04-06 at 10 02 29 AM

**Review by FSM1 — APPROVED** [review]

**Review by tanmoyAtb — APPROVED** [review]
Really smooth implementation 👍

**Pull request merged**